sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/microsoft/azure/messaging/servicebus/perf/core/ServiceBusStressOptions.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.messaging.servicebus.perf.core;
+
+import com.azure.perf.test.core.PerfStressOptions;
+import com.beust.jcommander.Parameter;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+/**
+ * Represents the command line configurable options for a performance test.
+ */
+@JsonPropertyOrder(alphabetic = true)
+public class ServiceBusStressOptions extends PerfStressOptions {
+
+    @Parameter(names = { "-mr", "--maxReceive" }, description = "MaxReceive messages")
```

Comment:
Why '-mr" and "-ms"? vs "-r" "--receive" and "-s", "--send"? It's a lot of characters to type.

Reply:
Good suggestion but we have a conflict. The parent class of test framework uses `-s`, we can use it here 
`@Parameter(names = { "-s", "--size" }, description = "Size of payload (in bytes)")`
We could probably use `-r` but to keep symmetry, between send and receive, I think we have to  `-mr and -ms` .